### PR TITLE
Anirud/final license header additions

### PR DESCRIPTION
**Pull request by anirudTT**

### CODE REVIEW

**Comment by anirudTT** [comment]
check this folder if the lc are correct :`model_demos/nlp_demos/falcon/utils`

File: model_demos/nlp_demos/falcon/utils/configuration_RW.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# Apache-2.0
```
Comment (milank94) [inline model_demos/nlp_demos/falcon/utils/configuration_RW.py:1]: I don't think that we need to include any additional headers to this file as the original authors have already included the appropriate header.

File: model_demos/nlp_demos/falcon/utils/model.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# Apache-2.0
```
Comment (milank94) [inline model_demos/nlp_demos/falcon/utils/model.py:1]:
```suggestion
# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
# SPDX-License-Identifier: Apache-2.0
```

File: model_demos/nlp_demos/falcon/utils/pybudify.py [hunk 3]
```diff
@@ -1,3 +1,5 @@
+# Apache-2.0
```
Comment (milank94) [inline model_demos/nlp_demos/falcon/utils/pybudify.py:1]:
```suggestion
# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
# SPDX-License-Identifier: Apache-2.0
```

File: model_demos/nlp_demos/falcon/utils/tt_modeling_RW_pad_masked_odkv.py [hunk 4]
```diff
@@ -1,3 +1,5 @@
+# Apache-2.0
```
Comment (milank94) [inline model_demos/nlp_demos/falcon/utils/tt_modeling_RW_pad_masked_odkv.py:1]:
```suggestion
# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
# SPDX-License-Identifier: Apache-2.0
```

File: model_demos/cv_demos/yolo_v3/pytorch_yolov3_holli.py [hunk 5]
```diff
@@ -1,3 +1,6 @@
+# SPDX-FileCopyrightText: Copyright (c) 2018 Olli Huotari
```
Comment (milank94) [inline model_demos/cv_demos/yolo_v3/pytorch_yolov3_holli.py:1]: This file was created by us. Only the source model file comes from the other author.

So this one can contain:
```python
# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
# SPDX-License-Identifier: Apache-2.0
```

File: model_demos/cv_demos/yolo_v3/pytorch_yolov3_holli_1x1.py [hunk 6]
```diff
@@ -1,3 +1,6 @@
+# SPDX-FileCopyrightText: Copyright (c) 2018 Olli Huotari
```
Comment (milank94) [inline model_demos/cv_demos/yolo_v3/pytorch_yolov3_holli_1x1.py:1]: This file was created by us. Only the source model file comes from the other author.

So this one can contain:
```python
# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
# SPDX-License-Identifier: Apache-2.0
```

File: model_demos/cv_demos/yolo_v3/pytorch_yolov3_tiny_holli.py [hunk 7]
```diff
@@ -1,3 +1,6 @@
+# SPDX-FileCopyrightText: Copyright (c) 2018 Olli Huotari
```
Comment (milank94) [inline model_demos/cv_demos/yolo_v3/pytorch_yolov3_tiny_holli.py:1]: This file was created by us. Only the source model file comes from the other author.

So this one can contain:
```python
# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
# SPDX-License-Identifier: Apache-2.0
```

**Comment by milank94** [comment]
The files in `cv_demos/yolo_v3/holli_src` will need to contain the header:
```python
# SPDX-FileCopyrightText: Copyright (c) 2018 Olli Huotari
# SPDX-License-Identifier: MIT
```

File: check_copyright_config.yaml [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+ignore:  # You can also select ignoring files here
+  perform_check: no  # Don't check files from that block
+  include:
+    - .github/*
+    - __pycache__/*
+    - /CONTRIBUTING.md
+    - /MAINTAINERS.md
+    - /model_demos/cv_demos/yolo_v3/holli_src/*
+    - /model_demos/nlp_demos/falcon/utils/*
```
Comment (milank94) [inline check_copyright_config.yaml:32]: From this directory, `model_demos/nlp_demos/falcon/utils/configuration_RW.py` should remain unchanged as it already has the source header.

The other files will contain the Tenstorrent header as they are our own generated files.

File: .github/workflows/benchmarking-workflow.yml [hunk 9]
```diff
@@ -0,0 +1,60 @@
+name: Validate SPDX License year and company mention in currently changed Files
```
Comment (milank94) [inline .github/workflows/benchmarking-workflow.yml:1]: I think this file should be renamed from `benchmarking-workflow.yml` to something more general or descriptive for this repo.

Comment (anirudTT) [reply .github/workflows/benchmarking-workflow.yml:1]: This was just for testing the old workflow -- currently in the benchmarking repo , will be removed before merging the PR

File: model_demos/nlp_demos/falcon/utils/configuration_RW.py (hunk 1 above)
Comment (anirudTT) [reply model_demos/nlp_demos/falcon/utils/configuration_RW.py:1]: remove this license header

**Pull request closed without merging**